libs/util/nestjs/kafka/project.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "../../../../node_modules/nx/schemas/project-schema.json",
   "name": "util-nestjs-kafka",
+  "$schema": "../../../../node_modules/nx/schemas/project-schema.json",
```

Comment:
is there a way to create alias to node module, instead of using that root ?

Reply:
Potentially, but this transformation has been performed by nx migration script so I would avoid any change for now.